scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -691,13 +691,13 @@ private[scalatest] class DashboardReporter(directory: String,
     //
     def toXml: String = {
       val buf = new StringBuilder
-      var testRecord: TestRecord = null
+      var testRecord: TestRecord | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.